csfieldguide/static/interactives/city-trip/js/city-trip.js

```diff
@@ -0,0 +1,642 @@
+const cytoscape = require('cytoscape');
+const noOverlap = require('cytoscape-no-overlap');
+const automove = require('cytoscape-automove');
+const Mathjs = require('mathjs');
+
+cytoscape.use(noOverlap);
+cytoscape.use(automove);
+var stopPathFinding = false;
+var stoppedMidExecution = false; // true if the stop button been clicked mid path finding.
+
+const STATUS_COMPLETE = gettext("complete!");
+const STATUS_READY = gettext("ready to go!");
+const STATUS_STOPPED = gettext("stopped");
+const STATUS_RUNNING = gettext("running");
+
+$(document).ready(function() {
```

Comment:
I think we can do better for phone screens

![image](https://user-images.githubusercontent.com/25916475/56100978-4d0baf80-5f73-11e9-996a-d783ef27a9ee.png)

Reply:
We have to fix the size of the container for the graph, so I don't think I can get it any better :( the user is able to scroll across but I know that is less than ideal. Had a talk to Jack about it.

Reply:
Can you not have a second smaller fixed-container size for these situations? We should at least add some sort of warning like 2d-shape-manipulations has

Reply:
Added warning